src/NuGetTrends.Spa/src/environments/environment.ts

```diff
@@ -3,10 +3,11 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
+  name: 'local',
   production: false,
   API_URL: 'https://localhost:5001',
   MAX_CHART_ITEMS: 6,
-  SENTRY_DSN: ''
+  SENTRY_DSN: 'https://85a592e835c64ca3a97d93776c12e947@sentry.io/1266321'
```

Comment:
Was this accidental or u want to publish the 'local' env with this DSN?

Reply:
It was accidental, but later I thought of leaving it. Not that I added the environment, it should not pollute the prod events. What do you think?

Reply:
Works for me